config/initializers/new_framework_defaults_6_1.rb

```diff
@@ -0,0 +1,67 @@
+# Be sure to restart your server when you modify this file.
+#
+# This file contains migration options to ease your Rails 6.1 upgrade.
+#
+# Once upgraded flip defaults one by one to migrate to the new default.
```

Comment:
First time I've noticed the comments were meant to do something - going through it now. There are a few that say ```# It's best enabled when your entire app is migrated and stable on 6.1.``` so I'll go for all the other ones that don't specify that

Reply:
ah on a side note: I also had to run "bundle update" to make the app work. But while that changed the Gemfile.lock it didn't change the actual Gemfile, so maybe I need to manually add the latest versions of the gems manually to the gemfile?

Reply:
This is how Gemfiles work. The Gemfile defines the gems, and can limit the versions (either to a specific version, or a range https://bundler.io/gemfile.html) and then the lockfile sets it to a specific version. When you run "bundle update" it will check to see if there are any newer versions of those gems that match the Gemfile.